app/scripts/lib/channels/notifier.js

```diff
@@ -18,9 +18,10 @@ define(function (require, exports, module) {
   var EVENTS = {
     COMPLETE_RESET_PASSWORD_TAB_OPEN: 'fxaccounts:complete_reset_password_tab_open',
     DELETE: 'fxaccounts:delete',
+    DEVICE_NAME_CHANGE: 'fxaccounts:device_name_change',
     PROFILE_CHANGE: 'profile:change',
     SIGNED_IN: 'internal:signed_in',
-    SIGNED_OUT: 'internal:signed_out'
+    SIGNED_OUT: 'fxaccounts:logout'
```

Comment:
> I'd like to hear more about this change and what behaviour it will cause.

My novice understanding of things as they currently stand in my WIP branch:

When triggered it will invoke [this notification handler](https://github.com/philbooth/gecko-dev/blob/a3f82e3bf0cf1f961571ebbe15623cbe5ea66543/services/fxaccounts/FxAccountsWebChannel.jsm#L246-L252), which checks the sent `uid` and then kicks off [the Sync sign-out shizzle](https://github.com/philbooth/gecko-dev/blob/a3f82e3bf0cf1f961571ebbe15623cbe5ea66543/services/fxaccounts/FxAccounts.jsm#L610-L645) if it matches the currently signed-in user. This includes a [request to the auth server to destroy the device](https://github.com/philbooth/gecko-dev/blob/a3f82e3bf0cf1f961571ebbe15623cbe5ea66543/services/fxaccounts/FxAccounts.jsm#L667-L669).

To me that looks like we possibly`*` make duplicate calls to the delete endpoint, which seems wrong. However all I did in that bit of browser code was replace the session destroy with a device destroy, so maybe there was already a duplicate call to session destroy in there before (which is not to say I won't fix it, it's definitely wrong if it happens).

`*`The reason I say possibly is because of [this `!localOnly` guard around the destroy call](https://github.com/philbooth/gecko-dev/blob/a3f82e3bf0cf1f961571ebbe15623cbe5ea66543/services/fxaccounts/FxAccounts.jsm#L623), which I don't fully understand yet. Once I've fixed the blocking issue in my WIP branch, I'll trace this scenario to see which path of execution is taken and whether the duplicated call is actually made or not.